.github/workflows/continuous-integration.yml

```diff
@@ -77,7 +77,7 @@ jobs:
       run: yarn build
 
     - name: Run tox
-      run: tox --parallel auto -e checkformatting,lint,tests,functests,bddtests
+      run: tox --parallel -e checkformatting,lint,tests,coverage,functests,bddtests
```

Comment:
Also removed the `auto` from this line in a separate commit. It's not needed: `auto` is the default